.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Unnecessary changes here that should be removed. Since you are using `Visual Studio Code`, maybe you want to include `.vscode` in `.gitignore`. Also, we suggest you use `IntelliJ` or `Eclipse` for development.